panel/widgets/stats_plot_card.py

```diff
@@ -0,0 +1,61 @@
+"""The StatsPlotCard enables the user to display a Dashboard KPI Card with
+
+- Title: The name or a short title
+- Value: Primary Value to display. For example an absolute value.
+- Value2: A secondary value. For example a percentage change.
+- Plot: A plot. One of line", "step", "area", "bar
+
+The card can be layout out as
+
+- a column (text and plot on top of each other) or
+- a row (text and plot after each other)
+
+The card enables performant streaming.
+The text section is responsive and resizes on window resize.
+"""
+import param
+from bokeh.models.sources import ColumnDataSource
+from panel.models.stats_plot_card import StatsPlotCard as _BkStatsPlotCard
+from panel.widgets.base import Widget
+
+
+class StatsPlotCard(Widget):
```

Comment:
Let's make this an indicator and move it to `/widgets/indicators.py`

Reply:
done